browserstack.json

```diff
@@ -4,15 +4,31 @@
 	"test_framework": "qunit",
 	"test_path": [
 		"test/index.html",
+		"test/startError.html",
 		"test/logs.html",
-		"test/amd.html"
+		"test/amd.html",
+		"test/autostart.html"
 	],
 	"browsers": [
 		"chrome_previous",
 		"chrome_current",
 		"firefox_previous",
 		"firefox_current",
+		"opera_previous",
 		"opera_current",
+		"safari_5_1",
+		{
+			"browser": "safari",
+			"browser_version": "6.0"
+		},
+		{
+			"browser": "safari",
+			"browser_version": "7.0"
+		},
+		{
+			"browser": "safari",
+			"browser_version": "7.1"
+		},
 		"safari_previous",
 		"safari_current",
```

Comment:
Basically, Safari major number corresponds to the OS X release, minor is bumped when engine is updated. Thus, 6.x is Mountain Lion, 7.x is Mavericks, 8.x is Yosemite, 9.x is El Capitan. 6.2 on Mountain Lion may have a different UI than 8.0 on Yosemite but WebKit used is the same.